stepactions/create-trusted-artifact/create-trusted-artifact.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: StepAction
+metadata:
+  name: create-trusted-artifact
+spec:
+  description: >-
+    This stepaction creates a trusted artifact. It does nothing if a .skip-trusted-artifacts file exists
+    in root folder.
+  image: quay.io/redhat-appstudio/build-trusted-artifacts:e02102ede09aa07187cba066ad547a54724e5cf4
+  env:
+    # Set HOME variable to something else than `/` to avoid 'permission denied' problems when writing files.
+    - name: HOME
+      value: /tekton/home
+  params:
+    - name: ociStorage
+      type: string
+      default: "empty"
+    - name: dataDir
+      type: string
+    - name: subdirectory
```

Comment:
Is there any reason to not include this in dataDir? And save needing this parameter

Reply:
fixed